app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/TableInlineEditRow/TableConfirmButtonsRow.js

```diff
@@ -0,0 +1,154 @@
+// This component is based on the component of the same name from patternfly-react.
+// It includes a fix for https://github.com/patternfly/patternfly-react/issues/535
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { createPortal } from 'react-dom';
+import { noop, debounce } from 'patternfly-react';
+import './closestPolyfill';
+import ConfirmButton from '../InlineEdit/ConfirmButton';
```

Comment:
Yeah THESE TWO!!  should be combined into a single function contained in here, cuz they're not used elsewhere 🤔

Reply:
oh wait... this was done because.... its how pfr does it... and they likely consume the buttons elsewhere?

Reply:
Yeah, the inline edit stuff should be excluded from review because it's a temporary copy-over of the pf-react components.